packages/react/src/utils/useUnmountAfterCloseAnimation.tsx

```diff
@@ -0,0 +1,50 @@
+import * as ReactDOM from 'react-dom';
+import { useAnimationsFinished } from './useAnimationsFinished';
+import { useEnhancedEffect } from './useEnhancedEffect';
+import { useLatestRef } from './useLatestRef';
+
+export function useUnmountAfterExitAnimation(parameters: useUnmountAfterExitAnimation.Parameters) {
+  const {
+    open,
+    animated,
+    animatedElementRef,
+    setMounted,
+    flushUpdatesSynchronously = false,
```

Comment:
This shouldn't actually be necessary. Think it was a leftover from the old select `alignOptionToTrigger` implementation.